components/org.wso2.carbon.identity.api.user.recovery/src/gen/java/org/wso2/carbon/identity/recovery/endpoint/dto/ReCaptchaPropertiesDTO.java

```diff
@@ -33,7 +36,19 @@ public void setReCaptchaEnabled(Boolean reCaptchaEnabled) {
     this.reCaptchaEnabled = reCaptchaEnabled;
   }
 
-  
+
+  /**
```

Comment:
Add method comments

Reply:
This is an auto generated file from api.identity.recovery.yaml[1]. Therefore, no manual changes were done.

[1] https://github.com/wso2-extensions/identity-governance/blob/master/components/org.wso2.carbon.identity.api.user.recovery/src/main/resources/api.identity.recovery.yaml